app/code/Magento/Catalog/Model/Product/Image/Cache.php

```diff
@@ -24,11 +27,27 @@ class Cache
     protected $themeCollection;
 
     /**
+     * @var ThemeCustomizationConfig
+     */
+    protected $themeCustomizationConfig;
```

Comment:
Adding new `protected` properties is expanding the contract of the class, which is not really desired in this case. Please change to `private`

Reply:
changed to private